shared-version.properties

```diff
@@ -0,0 +1,2 @@
+gradle.ext.blackDuckCommonVersion='62.0.0-SNAPSHOT'
```

Comment:
Should the BDC version be 61.2.3?

Reply:
Yes good catch

Reply:
Actually the latest blackduck-common isn't using the latest integration-bdio.
I will release Black Duck common 62.0., then update this.

Reply:
Updated to the now released blackduck-common:62.0.1